Plugins/BackgroundFetch/BackgroundFetchPlugin.cs

```diff
@@ -25,6 +25,11 @@ public BackgroundFetchPlugin() : base(true)
         private IDisposable _cancellationToken;
         private IGitUICommands _currentGitUiCommands;
 
+        private readonly PseudoSetting _warningForceWithLease = new PseudoSetting("Warning: be aware that the push force with lease feature should be used carefully when the periodic background fetch is enabled. Indeed, the protection brought by this feature could be bypassed if the background fetch has updated the remote branches. And you could loose the commits newly pushed by someone else. So be sure to refresh the revsion grid before doing a force push with lease.", textboxSettings: tb =>
```

Comment:
Few typos in the original text.

The text looks a bit verbose, what do you think of this variant?
```suggestion
        private readonly PseudoSetting _warningForceWithLease = new PseudoSetting("WARNING: be careful when force push with lease having the periodic background fetch enabled but chose not to auto-refresh after each fetch. You could lose new commits pushed by others to the remote branch. Be sure to refresh the revision grid before doing a force push with lease.", textboxSettings: tb =>
```

Reply:
updated.